ci/windows/build_samples.ps1

```
@@ -0,0 +1,68 @@
+$configure_command += " -DCMAKE_CUDA_STANDARD=$CXX_STANDARD"
+$configure_command += " -DCCCL_SOURCE_DIR=`"$REPO_ROOT`""
+if ($CUDA_ARCH) {
+    $configure_command += " -DCMAKE_CUDA_ARCHITECTURES=`"$CUDA_ARCH`""
+}
+if ($CMAKE_OPTIONS) {
+    $configure_command += " $CMAKE_OPTIONS"
+}
+
+Write-Host "=== Configure ==="
+Write-Host $configure_command
+Invoke-Expression $configure_command
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)